backend/src/test/java/codezap/member/service/MemberServiceTest.java

```diff
@@ -42,7 +53,13 @@ void signup() {
             Member member = MemberFixture.memberFixture();
             SignupRequest signupRequest = new SignupRequest(member.getName(), member.getPassword());
 
-            assertEquals(memberService.signup(signupRequest), 1L);
+            Long savedId = memberService.signup(signupRequest);
+
+            boolean isDefaultCategory = categoryRepository.existsByNameAndMember("카테고리 없음", member);
```

Comment:
```suggestion
            boolean existsDefaultCategory = categoryRepository.existsByNameAndMember("카테고리 없음", member);
```
변수명을 조금만 수정해보면 의미가 좀 더 명확해질 것 같아요